runners/console/index.ts

```diff
@@ -6,9 +6,9 @@ import { Playbook } from "../../engine/playbook";
 import { ConsolePlatform, AsyncProcess } from "./consoleInterfaces";
 import * as path from 'path';
 import * as fs from "fs-extra";
-import * as psList from "ps-list";
 import { ConsoleUtils } from "./consoleUtils";
 
+const { listProcess } = require("process-list");
```

Comment:
You need to add the package to the package.json file, don't you?
And the brackets are not needed I think.

Reply:
I have removed the brackets